### DISCUSSION TITLE
Regional Settings/Time Zones

### DISCUSSION THREAD

**Discussion by dunc1ca** [Freika/dawarich#412] [Ideas]
Is there a way to change the date/time format?  Here is Canada, we use yyyy-mm-dd.  Also, it would be nice to adjust the time zone in the settings in a drop down menu and have the capability that wherever on the map you are browsing, times are shown in local time.

**Comment by Freika** [comment]
Hi @dunc1ca, can you provide more context of what part of the interface you're referring to? Is it only the map datetime selector, or something else?

**Reply by dunc1ca** [reply]
I see the date in dd/mm/yyyy format when mousing over a point on the map.  Also, the visited cites on the map margin show the date in this format.

**Comment by redge76** [comment]
Same here. I guess the locale setting are pulled from the browser headers (accept-language ...)
My timeformat in Google has a US locale (AM/PM) . In edge I have a CET timezone and time firmat (24h)
Is there some settings to force some locale and not rely on browser settings ?

**Reply by Freika** [reply]
For now there isn't, it might be added later on though.

**Reply by ignaty8** [reply]
I think it might be done a bit differently with map points. 
<img width="222" height="151" alt="Screenshot 2025-12-23 at 00 48 01" src="https://github.com/user-attachments/assets/ceecd6e2-3fa2-4c23-8010-d51e1f6484a2" />
Here's an example where I have two formats show up (locale set to UK). Might be Leaflet is misconfigured?

**Reply by Freika** [reply]
No, it's just that the input in date selection form is based on your browser locale settings and in leaflet it's hardcoded

**Comment by esammier** [comment]
+1 for the timezone selection. My Dawarich instance is hosted in a timezone 1 hour ahead of mine, and I have noticed that if I check the map after 23:00, it shows my current location in the next day.
I'm logging with GPSLogger using the template from the [docs](https://dawarich.app/docs/tutorials/track-your-location#gps-logger), which as I understand it reports UTC timestamps, but maybe Dawarich expects local time?
